converters/fromZigbee.js

```diff
@@ -4759,6 +4759,15 @@ const converters = {
             }
         },
     },
+    ikea_voc_index: {
```

Comment:
Since this is IKEA specific, you should move this to devices/ikea.js where the other ikea specific from/toZigbee converters are.

Reply:
Thank you, fixed